### PR TITLE
PXC-3592: Make wsrep_provider and wsrep_notify_cmd read only

### PR DESCRIPTION
**Pull request by dutow**

### CODE REVIEW

**Comment by dutow** [comment]
https://pxc.cd.percona.com/view/PXC%205.6/job/pxc-5.6-param/121/

File: mysql-test/suite/galera/r/galera_ist_rsync.result [hunk 1]
```diff
@@ -39,7 +38,7 @@ INSERT INTO t1 VALUES ('node1_to_be_rollbacked_after');
 INSERT INTO t1 VALUES ('node1_to_be_rollbacked_after');
 INSERT INTO t1 VALUES ('node1_to_be_rollbacked_after');
 INSERT INTO t1 VALUES ('node1_to_be_rollbacked_after');
-Loading wsrep provider ...
+SET GLOBAL wsrep_cluster_address = 'gcomm://127.0.0.1:13002';;
```
Comment (kamil-holubicki) [inline mysql-test/suite/galera/r/galera_ist_rsync.result]: 1. double semicolon (multiple occurences)
2. Won't having an explicit port number here cause problems when running tests in parallel?

Both will be solved by fixing galera_load_provider.inc

File: mysql-test/suite/galera/include/galera_load_provider.inc [hunk 2]
```diff
@@ -1,8 +1,3 @@
---echo Loading wsrep provider ...
-
---disable_query_log
```
Comment (kamil-holubicki) [inline mysql-test/suite/galera/include/galera_load_provider.inc:3]: this line was removed unnecessarily.

**Review by kamil-holubicki — CHANGES_REQUESTED** [review]

File: mysql-test/suite/galera/include/galera_load_provider.inc (hunk 2 above)
Comment (dutow) [reply mysql-test/suite/galera/include/galera_load_provider.inc:3]: Good catch and this causes port 13002 to appear, I'll fix this.

Comment (dutow) [reply mysql-test/suite/galera/include/galera_load_provider.inc:3]: Fixed.

**Review by kamil-holubicki — APPROVED** [review]
lgtm

**Pull request merged**